README.md

```diff
@@ -59,6 +59,23 @@ Options:
 
 `nancy` currently works for projects that use `dep` or `go mod` for dependencies.
 
+#### Docker usage
+
+<p align="center">
+    <img src="https://github.com/sonatype-nexus-community/nancy/blob/master/docs/images/nancy_docker.png" width="350"/>
+</p>
+
+`nancy` now comes in a boat! For ease of use, we've dockerized `nancy`. To use our Dockerfile:
+
+`go list -m all | docker run -i sonatypecommunity/nancy:latest`
```

Comment:
This is good and basically solves for not waiting to necessarily install Nancy other than via docker.

Maybe it’s not this pr but should we also provide build images too?? That are based on golang base images and have Nancy pre-installed??

Thinking then on any ci that uses docker images to do builds (concourse, GitHub actions, etc) it would allow you as developer to skip the step of building it yourself.

Reply:
The original repo `docker-nancy` had it built as two stage, builder image and then the image for running it. I'm not SUPER worried about providing a build image since we would be providing images for it on Dockerhub?

Reply:
Oh I see what you mean now. That's not a bad idea! @bhamail and I had talked a bit about that, specifically a CircleCI orb for Nancy, and like what's our goal with this? Is it so that people can just use Nancy, or is it so they can build stuff AND nancy is there?

Reply:
Agree that it could be something else we do outside of this. You might talk to @bhamail and spitball, we did a Nancy orb for CircleCI already, but it could go any number of ways!

Reply:
@DarthHater So you already did the orb?? Where does that live?? Should we just add it to this repo??